kratos/tests/test_variable_redistribution.py

```diff
@@ -6,45 +6,26 @@
     have_fsi = False
 
 import KratosMultiphysics.KratosUnittest as UnitTest
-
-from os import remove
-
-class WorkFolderScope:
-    def __init__(self, work_folder):
-        self.currentPath = os.getcwd()
-        self.scope = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)),work_folder))
-
-    def __enter__(self):
-        os.chdir(self.scope)
-
-    def __exit__(self, type, value, traceback):
-        os.chdir(self.currentPath)
+import KratosMultiphysics.kratos_utilities as KratosUtilities
 
 @UnitTest.skipUnless(have_fsi,"Missing required application: FSIApplication")
 class VariableRedistributionTest(UnitTest.TestCase):
 
     def setUp(self):
-        self.domain_size = 2
         self.input_file = "redistribution_test"
         self.work_folder = "auxiliar_files_for_python_unittest/RedistributionTest"
 
         self.redistribution_iterations = 100
-        self.redistribution_tolerance = 1e-7
+        self.redistribution_tolerance = 1e-8
 
         self.check_tolerance = 1e-6
         self.print_output = False
 
         self.mapped_function = None
 
     def tearDown(self):
-        self.deleteOutFile(self.input_file+'.time')
-
-    def deleteOutFile(self,filename):
-        with WorkFolderScope(self.work_folder):
-            try:
-                remove(filename)
-            except FileNotFoundError as e:
-                pass
+        with UnitTest.WorkFolderScope(self.work_folder,__file__):
+            KratosUtilities.DeleteFileIfExisting(self.input_file + '.time')
```

Comment:
May be we can do this?

```suggestion
            KratosUtilities.DeleteTimeFiles('.')
```